### PR TITLE
Mandatory copyright agreement

### PR DESCRIPTION
**Pull request by achow101**
All contributions to the repository will agree to use the MIT license.

### CODE REVIEW

**Comment by laanwj** [comment]
Maybe "unless specified otherwise in contrib/debian/copyright or at the top of the file itself". There are some exceptions, which may be acceptable. This is more meant as a default policy to avoid the flurry of #8784, #8702, #8701, #8700, #8676, from ever having to be done again.

**Comment by achow101** [comment]
@laanwj Done

**Comment by luke-jr** [comment]
ACK

**Comment by laanwj** [comment]
ACK

**Comment by btcdrak** [comment]
ACK

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -198,3 +198,8 @@ Release Policy
 --------------
 
 The project leader is the release manager for each Bitcoin Core release.
+
+Copyright
+---------
+
+By contributing to this repository, you agree to license your work under the MIT license unless specified otherwise in `contrib/debian/copyright` or at the top of the file itself. Any code contributed where you are not the original author must contain its license header.
```
Comment (MarcoFalke) [inline CONTRIBUTING.md]:
```vim
:%s/code/work/g
```

We also host images.

Nit: Can you make this 80 chars wide?

Comment (MarcoFalke) [inline CONTRIBUTING.md]: Also append to the last sentence maybe:

```
..., the original author and source.

**Comment by MarcoFalke** [comment]
Thanks! Concept ACK

**Comment by achow101** [comment]
@MarcoFalke done

**Pull request merged**